src/Microsoft.AspNetCore.Razor.Runtime/Runtime/TagHelpers/TagHelperDescriptorFactory.cs

```diff
@@ -153,7 +156,7 @@ public TagHelperDescriptorFactory(bool designTime)
                         typeName,
                         assemblyName,
                         attributeDescriptors,
-                        requiredAttributes: Enumerable.Empty<string>(),
+                        requiredAttributes: Enumerable.Empty<TagHelperRequiredAttributeDescriptor>(),
```

Comment:
Will this break `RazorTooling` or tooling? I was expecting to see an _additional_ property containing the parsed details.